components/billing-summary-card.tsx

```diff
@@ -25,8 +25,7 @@ export function BillingSummaryCard() {
           <div className="flex justify-between items-center">
             <div className="text-sm font-medium">Billing Period</div>
             <div className="text-lg font-semibold text-foreground">
-              {startOfMonth.toLocaleDateString(undefined, { dateStyle: "long" })} -{" "}
-              {endOfMonth.toLocaleDateString(undefined, { dateStyle: "long" })}
+              {startOfMonth.toLocaleDateString('en-CA').replace(/-/g, '/')} - {endOfMonth.toLocaleDateString('en-CA').replace(/-/g, '/')}
```

Comment:
Do we really want to force the formatting? IMHO people want to see the format they are used to see. e.g. everyone I know want to see <Day>. <Month>. <Year> e.g. 12. 3. 2025
But we can force it if necessary.